data-es2016plus.js

```diff
@@ -806,6 +806,40 @@ exports.tests = [
           firefox2: false,
           firefox52: true,
           opera10_50: false,
+          safari10_1: true,
+          duktape2_0: false,
+          graalvm: true,
+        }
+      },
+      {
+        name: 'async arrow functions, classes',
+        exec: function () {/*
+          class C {
+            a = async () => { return await Promise.resolve("foo"); }
```

Comment:
this syntax is class fields, which is stage 3, and as far as i know, only shipped in chrome.

A test for async arrow functions shouldn't need class fields; if it's a class fields bug, the test should go there instead.

Reply:
Trying to figure this out, will come back to you soon :+1:

Reply:
@ljharb just so you know, this is the bug we are trying to get rid of: https://bugs.webkit.org/show_bug.cgi?id=166879

Reply:
@Titozzz so just use something like the first example from the tracker - it does not use class fields.

Reply:
You mean inside the compat test @zloirock?

Reply:
@Titozzz I mean that you can use something like [this](https://bugs.webkit.org/show_bug.cgi?id=166879#c0) here, in async functions tests, without usage class fields which can't be used here.

Reply:
I have updated the PR (but not 100% sure about my test), could you please tell me what you think?

Reply:
If it can detect this bug - LGTM.

Reply:
We just copy-pasted the code in each browser, and it does detect the bug when needed :+1: